openlibrary/macros/OLID.html

```diff
@@ -0,0 +1,15 @@
+$def with (q="")
+
+$ edition = query_param('edition', q)
+$ result = edition.olid_link(q)
+$ olid = edition.key.split("/")[-1]
+$ title = edition.title
+
+$if(olid == q):
+    <a href="$edition.key">$title</a>
+    <h6>$edition.get_authors()</h6>
```

Comment:
We've defined `edition` as `edition = query_param('edition', q)` but this doesn't actually get an edition so none of the other code works or makes sense here.